daemon/daemon_unix.go

```diff
@@ -186,7 +190,10 @@ func parseSecurityOpt(container *container.Container, config *containertypes.Hos
 
 		switch con[0] {
 		case "label":
-			labelOpts = append(labelOpts, con[1])
+			if selinuxConvert {
+				opt = strings.Replace(opt, ":", "=", 1)
```

Comment:
Can you provide some extra detail/context to this change (maybe in the main body of the PR/in a PR comment)?